### PR TITLE
add banner session, with index.html and _2_banner.scss file

### LINKED ISSUE #4: banner
image

### PR DESCRIPTION
**Pull request by vominhmanh**
close #4

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,6 +55,32 @@
             </nav>
         </div>
     </header>
+
+    <div class="align d-lg-none d-md-none d-sm-none"></div>
+
+    <banner class="header-banner shadow-lg">
```
Comment (Tuanhdhapo) [inline index.html:61]: dùng background

File: index.html [hunk 2]
```diff
@@ -55,6 +55,32 @@
...
+        <div id="carouselExampleIndicators" class="carousel slide" data-ride="false">
+            <ol class="carousel-indicators">
+                <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
+            </ol>
+            <div class="carousel-inner">
+                <div class="carousel-item active">
+                    <img class="d-block w-100" src="images/hapo_learn_banner.png" alt="First slide">
+
```
Comment (Tuanhdhapo) [inline index.html:69]: thừa enter line

File: scss/_2_banner.scss [hunk 3]
```diff
@@ -0,0 +1,79 @@
+.header-banner {
+  @media only screen and (max-width: 575.98px) {
+    .header-banner-title {
+      position: relative;
+      background-color: $gray;
+      padding: 5% 10%;
```
Comment (Tuanhdhapo) [inline scss/_2_banner.scss:6]: dùng px

**Review by hapo-nghialuu — APPROVED** [review]

**Comment by hapo-nghialuu** [comment]
LGTM

**Pull request merged**